### PR TITLE
Update Most Builds to VL+PLMC

### PR DESCRIPTION
**Pull request by bcaddy**
# Summary

Updated all builds, except cosmology, to use the Van Leer (VL) integrator and Piecewise Linear Method in the Characteristic variables (PLMC) reconstruction.

I'd like @helenarichie and @ojwg to take a look to make sure I'm not breaking any of their builds.

### CODE REVIEW

File: builds/make.type.hydro [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 DFLAGS    += -DMPI_CHOLLA
 DFLAGS    += -DPRECISION=2
-DFLAGS    += -DPPMC
+DFLAGS    += -DPLMC
 DFLAGS    += -DHLLC
 
 # Integrator
-DFLAGS    += -DSIMPLE
-#DFLAGS    += -DVL
+# DFLAGS    += -DSIMPLE
```
[the remark this answers is not in the document]
Comment (bcaddy) [reply builds/make.type.hydro]: done

**Comment by evaneschneider** [comment]
Just to double-check -- were you able to confirm that the spherical collapse test is actually collapsing now? It was working when I initially added the system test, but when @evazlimen documented it last August it was not.

**Comment by bcaddy** [comment]
I compared slices of the old and new fiducial data and there were qualitatively very similar; ie a big dot in the middle. I thought the old fiducial data was definitely correct but I’m not totally sure.

**Comment by evaneschneider** [comment]
Sounds good. If you could add a plot of the new slice data to the documented examples page, that would be great (ultimately we will need to do this for all the examples, but that's a longer-term project).

**Comment by bcaddy** [comment]
Sure. I assume you want a higher resulting plot like 128^3 or 256^3 rather than the low res test version?

**Comment by evaneschneider** [comment]
Yes, I think specifically the plot that results from using the example parameter file (which is 256^3).

**Review by helenarichie — COMMENTED** [review]
Looks good to me!

**Pull request merged**